apps/system/test/marionette/app_authentication_dialog_test.js

```diff
@@ -76,4 +76,28 @@ marionette('App Authentication Dialog',
       return !authDialog.displayed();
     });
   });
+
+  test('user can login', function() {
```

Comment:
I wonder if this should just be squashed into the previous test-case? Seems it's better to actually login then to just unprotect the URL?